dts/bindings/led/issi,is31fl3194.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2022 Benjamin Björnsson <benjamin.bjornsson@gmail.com>.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Lumissil is31fl3194 LED Driver
+
+compatible: "issi,is31fl3194"
+
+include: ["i2c-device.yaml", "led-controller.yaml"]
+
+child-binding:
+    description: TI LP503x LED controller node
```

Comment:
I don't think it is the right description :)

Reply:
Yeah, that doen't look right... Thank you!